my-sms/spec/decorators/student_decorator_spec.rb

```diff
@@ -6,7 +6,7 @@
   it 'formats a full name with title, first name, middle name, last name' do
     student = build(
       :student,
-      title: 'Mr',
+      title: FactoryBot.build(:title, name: 'Mr'),
```

Comment:
I would normally move the `title` here to a separate variable/let binding. Something like the following:

```ruby
subject { student.decorate }
let(:title) { create :title }
let(:student) { create :student, title: title, ... }

it "formats full name" do
  expect(subject.full_name).to eq "..."
end
```

Reply:
Fixed in fac18efd3fd5586517307c55737ab9d93f8b5dc9